apps-api/backoffice/src/modules/admin/election/repository.ts

```diff
@@ -447,37 +439,132 @@ export class AdminElectionRepository {
     )
   }
 
-  async upsertElectionOnsiteResult(result: { candidateId: string; votes: number }[]) {
+  async countElectionEligibleVoters(electionId: string, type: EligibleVoterType) {
+    return fromRepositoryPromise(
+      this.prismaService.electionEligibleVoter.count({
+        where: {
+          electionId,
+          type,
+        },
+      })
+    )
+  }
+
+  private async upsertElectionResult(
```

Comment:
Please detect invalid candidateId as invalid ballot card and store it in database for auditing too